styles/components/_connect-wallet.scss

```diff
@@ -94,6 +106,17 @@
           }
         }
       }
+      .auth-tip {
+        padding: 10px 0;
+        margin: 0 2rem;
```

Comment:
why mix `px` & `rem`?

Reply:
i just copied the style of this element `.account-item` in  [_connect-wallet.scss#L97](https://github.com/kodadot/nft-gallery/blob/main/styles/components/_connect-wallet.scss#L97) so the `.auth-tip` i added will keep the same spacing with it.
should i both change these two?

Reply:
you can safely remove `padding: 10px 0;` since we use `pl-5 pt-2 pb-2`